HISTORY.md

```diff
@@ -1,3 +1,11 @@
+## v4.0.0
+
+- multi-stage build support <https://docs.docker.com/develop/develop-images/multistage-build/>
+- switch to `node:8.9.4` base image for builder (same Debian base as before, but with Node 8 preinstalled)
+- Meteor 1.6.0.1 preinstalled
+- Mongo 3.4.11 (still optional at Reaction build time)
```

Comment:
Update this note to state that Mongo is removed.

Please add a notes here and to README about how to run a demo Reaction using the new `docker-compose-demo.yml` file (after name change of that file.)